app/grandchallenge/reader_studies/signals.py

```diff
@@ -40,6 +42,7 @@ def update_image_permissions(instance, action, reverse, model, pk_set, **_):
     op = assign_perm if "add" in action else remove_perm
 
     for rs in reader_studies:
+        op("view_image", rs.editors_group, images)
```

Comment:
Did the editors of a reader study not yet have permission to view the images before this PR?

Reply:
No - this was a new thing we added in #1573 but I missed the image permissions. No one complained though so maybe they were still using the same workflow?